mobile_app_connector/models/partner_communication.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from odoo import models, fields, _, api
+
+
+class CommunicationJob(models.Model):
+    _inherit = "partner.communication.job"
+
+    firebase_registration_exists = fields.Boolean(
+        compute='_compute_firebase_registration_exists')
+
+    mobile_notification_send = fields.Boolean("Send mobile notification", default=False)
+    mobile_notification_auto_send = fields.Boolean("Auto send")
+    mobile_notification_title = fields.Char("Title")
+    mobile_notification_body = fields.Char("Body")
+
+    mobile_notification_destination = fields.Selection([
+        ('MyHub', 'My Hub'),
+        ('Letter', 'Letter'),
+        ('Donation', 'Donation'),
+        ('Prayer', 'Prayer'),
+    ], "Destination", default="MyHub")
+
+    mobile_notification_topic = fields.Selection([
+        ('general_notification', 'General channel'),
+        ('child_notification', 'Child channel'),
+        ('spam', "None (bypass user's preferences)")
+    ], "Topic", default='general_notification')
+
+    mobile_notification = fields.Many2one("firebase.notification")
```

Comment:
Convention for relational fields naming is to suffix them with _id or _ids:
```suggestion
    mobile_notification_id = fields.Many2one("firebase.notification")
```